cardano-config/src/Cardano/TracingOrphanInstances/Common.hs

```diff
@@ -70,3 +76,196 @@ deriving newtype instance FromJSON EpochNo
 instance ToObject Void where
   toObject _verb x = case x of {}
 
+-- Common failures
+
+instance ToObject (PredicateFailure (DELEG c)) where
```

Comment:
These looks great, but they are not common failures, they are very much specific to the Shelley ledger.

Reply:
Yep my mistake here. Got confused with the types.